test/puppetlabs/puppetdb/config_test.clj

```diff
@@ -57,30 +57,31 @@
 
 (deftest database-configuration
   (testing "database"
-    (testing "validate-db-settings"
-      (let [cfg {:database {:classname "foo" :subprotocol "bar" :subname "baz"}}]
-        (is (= cfg (validate-db-settings cfg))))
-      (is (thrown+-with-msg? [:type ::conf/cli-error]
-                             #"database configuration is now required.* classname = .* subprotocol = .* subname = .*"
-                             (validate-db-settings {})))
-      (let [settings [:classname :subprotocol :subname]]
-        (doseq [x settings]
-          (let [other-settings (remove #(= % x) settings)
-                cfg-rx (re-pattern
-                        (apply str "database configuration is now required.*"
-                               (map #(str " " (name %) " = .*")
-                                    other-settings)))]
-            ;; Make sure the message contains settings for all but x.
-            (is (thrown+-with-msg? [:type ::conf/cli-error]
-                                   cfg-rx
-                                   (validate-db-settings {x "foo"})))
-            ;; Make sure the message doesn't contain a setting for x.
-            (is (try+
-                 (validate-db-settings {:database {x "foo"}})
-                 true
-                 (catch [:type ::conf/cli-error] {:keys [message]}
-                   (not (re-find (re-pattern (str " " (name x) " = "))
-                                 message)))))))))
+    (let [cfg {:database {:classname "org.postgresql.Driver"
```

Comment:
@ajroetker what did you have in mind?  Since HSQLDB support's gone, I think it should (currently) be treated just like any other invalid setting, and so may be covered by the "Bad :classname or :subprotocol" test.